view/components/main.html

```diff
@@ -15,7 +15,7 @@
                 <span>Listening ... </span>
                 <img src="../assest/icons/load.gif" alt="listening to you. Please keep speaking.." id="loader" srcset="">
             </div>
-            <img src="../assest/icons/mic.png" alt="Click to Start/Stop speaking" id="button" style="height: 40px;margin-left: 15px;" data-ng-click="toggleStartStop()" />
+            <img src="../assest/icons/mic.png" alt="Click to Start/Stop speaking" id="button" style="height: 40px;margin-left: 15px;" data-ng-click="toggleStartStop(1)" />
```

Comment:
please note the folder structure. its not _assest_, rather _assets_

Reply:
I think, it will itself get changed during the merge with the master branch, changing it now may create conflicts. So should I change or let it be how it is?

Reply:
oh! sure... let it be.